FUNDING_REQUEST.md

```diff
@@ -0,0 +1,53 @@
+# Requesting funding for Servo-related activities
+
+All use of Servo project funds must be approved by the [Servo TSC](governance/tsc/README.md).
+To make a funding request, create a new Zulip topic in the [#tsc channel](https://servo.zulipchat.com/#narrow/channel/500774-tsc/)
+that contains whatever context is necessary to evaluate the request.
```

Comment:
Should we request to create an issue on this repository (maybe we could have a special label for that issue) so we can keep a record of all the requests and if they were approved or not?

Starting a thread on #tsc seems good to, but also a GitHub issue with all the details.

Maybe we should even open an issue for the ongoing things, so we keep track there about the status and also modify it as needed (or close it) when the time comes.

Reply:
That sounds helpful!

Reply:
This is great!

Reply:
I guess we can merge this then.